src/core/MythosArea.ttslua

```diff
@@ -1,4 +1,14 @@
 local playAreaApi = require("core/PlayAreaApi")
+local tokenSpawnTracker = require("core/token/TokenSpawnTrackerApi")
+
+local ENCOUNTER_DECK_AREA = {
+  upperLeft = { x = 0.9, z = 0.42 },
+  lowerRight = { x = 0.86, z = 0.38 },
+}
+local ENCOUNTER_DISCARD_AREA = {
+  upperLeft = { x = 1.62, z = 0.42 },
+  lowerRight = { x = 1.58, z = 0.38 },
+}
```

Comment:
Actually, the ones in Global should be removed in favor of these :)  These decks belong to the Mythos area, and the draws should be its responsibility.  We can replace the Global access to them with an API on the mythos area to at least get the decks, and possibly handle the draw.

At the very least these are local, while the Global is, uhm, global... which makes the Global entries fragile if we move this panel.

Reply:
I don't mind where the values end up, but they should be in one place imo.
The "global" variables in the global object are still scoped to the global object and can only be accessed via `getVar` and `setVar` (not by accidentally reusing the names).

Reply:
Right, but what I mean is that the values in Global are table-space, not relative to the panel.  So if we were to move the panel as part of a table reorganization, those values would break (or at least need to be updated).

Reply:
And FWIW I agree that they should be in a single place.  But I don't want to perpetuate the use of getVar(), which is bad for all sorts of reasons, and refactoring the current encounter draw out of Global is way beyond what we have time for with the release target.

If you feel super strong about it I'll reference the vars in Global because it's the only reasonable option, but that's just replacing one bit of bad code with another and isn't improving anything.  Will leave this for you to resolve or not, let me know.

Reply:
Let's just keep this as is and maybe move variables later if needed